moto/rds2/models.py

```diff
@@ -229,17 +215,10 @@ def default_storage_type(iops):
     def default_allocated_storage(engine, storage_type):
         return {
             "aurora": {"gp2": 0, "io1": 0, "standard": 0},
+            "aurora-mysql": {"gp2": 20, "io1": 100, "standard": 10},
+            "aurora-postgres": {"gp2": 20, "io1": 100, "standard": 10},
```

Comment:
This should be `aurora-postgresql` (missing `ql`). Also same for the port above.

Reply:
Thanks :)